scanpy/external/pl.py

```diff
@@ -234,3 +236,90 @@ def sam(
         if colorbar:
             plt.colorbar(cax, ax=axes)
     return axes
+
+
+@_doc_params(show_save_ax=doc_show_save_ax)
+def wishbone_marker_trajectory(
+    adata: AnnData,
+    markers: List[str],
```

Comment:
```suggestion
    markers: Collection[str],
```

Markers needs to be `Iterable` and `Sized` (i.e. have `len()`). You can import `Collection` from `typing`